### ISSUE TITLE
Is Pose Loss Used for Backpropagation as Described in the Paper？

### ISSUE THREAD

**Issue by boss-create** [alexandor91/se3-equi-graph-registration#11]
Hello, your work is very good. And I noticed that the paper mentions using pose_loss, which includes rotation and translation losses, for computation. However, upon reviewing the code, it seems that these losses are not actually used for backpropagation. Could you please clarify if I have misunderstood something?

```
# part of code in 3DMatchTrainval 
# Combine the normalized losses if needed
        total_loss = corr_loss_mean #  + ssim_loss_mean  ### 

        # Combine pose and correspondence losses
        loss = total_loss #+ beta * corr_loss  
        torch.autograd.set_detect_anomaly(True)
        
        loss.backward()
```

**Comment by alexandor91** [comment]
Thank you for being so interested in our work. You're right—the pose loss should indeed be enabled. I had temporarily removed the pose loss (rotation + translation) during ablation testing with correspondence-based training and simply forgot to re-enable it.

I've now corrected this in the latest commit. I appreciate your careful review of the code and pointing out the issue. Feel free to use any loss combination that the code provides for your experiment, please.
